static/app/views/issueDetails/groupRelatedIssues/index.spec.tsx

```diff
@@ -0,0 +1,72 @@
+import {OrganizationFixture} from 'sentry-fixture/organization';
+import {RouterFixture} from 'sentry-fixture/routerFixture';
+
+import {render, screen, waitFor} from 'sentry-test/reactTestingLibrary';
+
+import {GroupRelatedIssues} from 'sentry/views/issueDetails/groupRelatedIssues';
+
+describe('Related Issues View', function () {
+  let relatedIssuesMock: jest.Mock;
+  let issuesInfoMock: jest.Mock;
+  const router = RouterFixture();
+
+  const organization = OrganizationFixture();
+  const orgSlug = organization.slug;
+  const groupId = '12345678';
+  const relatedGroup = '15';
```

Comment:
We're going to check for related issues for 12345678 and group number 15 will be related to it.